viewing_party/main.py

```diff
@@ -0,0 +1,165 @@
+def create_movie(title, genre, rating):
+      # if all three parameters are truthy, return a dict with key val pairs
+       #return a dictionary with 3 keys title, genre, rating 
+    #else title, genre, or rating is falsey
+        #return None
+    movie_dict = {}
+    movie_args = []
+    movie_args.append(title)
+    movie_args.append(genre)
+    movie_args.append(rating)
+    for i in movie_args:
+        if not i:
+            return None
+            
+    
+    movie_dict['title'] = movie_args[0]
+    movie_dict['genre'] = movie_args[1]
+    movie_dict['rating'] = movie_args[2]
+    return movie_dict
+   
+
+def add_to_watched(user_data, movie):
+    # user_data = {'watched':[]} empty list means no watched movies in their watched[]
+   
+    user_data['watched'].append(movie)
+    return user_data
+   
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    
+    return user_data
+
+def watch_movie(user_data, title):
+    
+    for movie in user_data['watchlist']:
+        if title == movie['title']:
+            user_data['watched'].append(movie)
```

Comment:
Consider using `add_to_watched`
```suggestion
            add_to_watched(user_data, each_movie)
```